src/component/page/bruker-detaljer/deltaker-detaljer/endre-deltaker-modal/AarsakSelector.tsx

```diff
@@ -33,8 +36,24 @@ export const AarsakSelector = ({ tittel, onAarsakSelected }: AarsakSelectorProps
 			<AarsakRadio aarsakType={DeltakerStatusAarsakType.TRENGER_ANNEN_STOTTE} />
 			<AarsakRadio aarsakType={DeltakerStatusAarsakType.IKKE_MOTT} />
 			<AarsakRadio aarsakType={DeltakerStatusAarsakType.UTDANNING} />
+			{ skalViseOppfyllerIkkeKrav && <AarsakRadio aarsakType={DeltakerStatusAarsakType.OPPFYLLER_IKKE_KRAVENE} >
```

Comment:
Det har ikke så mye å si, men her kan vi vel vurdere å ha et eget komponent for Aarsak med beskrivelse, siden de er helt like? Man kan sende inn type, og valgt aarsak som props, og om de er lik så vis beskrivelse. Da slipper vi vel en del sånn dobbelt opp som idene og disse `visBeskrivelse` predikatene.